File: LeetCode_Problem_9_Solution_1.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0:
            return False

        # convert from int to string
        str_num = str(x)
        copy = str_num

        # if we reverse one of the copies and both are still equal, then return
        # true
        copy = copy[::-1]

        if copy == str_num:
            return True
        else:
            return False

File: LeetCode_Problem_9_Solution_1.java
class Solution {
    public boolean isPalindrome(int x) {
        if (x < 0)
            return false;

        // convert from int to string
        String str_num = Integer.toString(x);
        String copy = str_num;

        // if we reverse one of the copies and both are still equal, then return
        // true
        copy = new StringBuilder(copy).reverse().toString();

        if (copy.equals(str_num))
            return true;

        else
            return false;
    }
}